### ISSUE TITLE
Ruby 3.3 support?

### ISSUE THREAD

**Issue by waghanza** [camping/camping#164]
Hi @karloscarweber ,

Ruby 3.3 requires [`racc`](https://github.com/ruby/racc)

I can not start-up https://github.com/the-benchmarker/web-frameworks/blob/master/ruby/camping/camp.rb

I have
```ruby
/usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require': cannot load such file -- racc/parser.rb (LoadError)
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /usr/local/bundle/gems/kdl-1.0.3/lib/kdl/kdl.tab.rb:7:in `<top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /usr/local/bundle/gems/kdl-1.0.3/lib/kdl.rb:8:in `<top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /usr/local/bundle/gems/camping-3.1.3/lib/camping/gear/kuddly.rb:5:in `<top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /usr/local/bundle/gems/camping-3.1.3/lib/camping/loads.rb:15:in `<top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /usr/local/bundle/gems/camping-3.1.3/lib/camping.rb:1:in `<top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `require'
        from /usr/local/lib/ruby/3.3.0/bundled_gems.rb:74:in `block (2 levels) in replace_require'
        from /usr/local/bundle/gems/camping-3.1.3/bin/camping:5:in `<top (required)>'
        from /usr/local/bundle/bin/camping:25:in `load'
        from /usr/local/bundle/bin/camping:25:in `<top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundler/cli/exec.rb:58:in `load'
        from /usr/local/lib/ruby/3.3.0/bundler/cli/exec.rb:58:in `kernel_load'
        from /usr/local/lib/ruby/3.3.0/bundler/cli/exec.rb:23:in `run'
        from /usr/local/lib/ruby/3.3.0/bundler/cli.rb:451:in `exec'
        from /usr/local/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor/command.rb:28:in `run'
        from /usr/local/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
        from /usr/local/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor.rb:527:in `dispatch'
        from /usr/local/lib/ruby/3.3.0/bundler/cli.rb:34:in `dispatch'
        from /usr/local/lib/ruby/3.3.0/bundler/vendor/thor/lib/thor/base.rb:584:in `start'
        from /usr/local/lib/ruby/3.3.0/bundler/cli.rb:28:in `start'
        from /usr/local/lib/ruby/gems/3.3.0/gems/bundler-2.5.3/exe/bundle:28:in `block in <top (required)>'
        from /usr/local/lib/ruby/3.3.0/bundler/friendly_errors.rb:117:in `with_friendly_errors'
        from /usr/local/lib/ruby/gems/3.3.0/gems/bundler-2.5.3/exe/bundle:20:in `<top (required)>'
        from /usr/local/bin/bundle:25:in `load'
        from /usr/local/bin/bundle:25:in `<main>'
```

Regards,

**Comment by karloscarweber** [comment]
Ruby 3.3 support is targeted for Q1 of this year.

**Comment by karloscarweber** [comment]
`racc` as a dependency will arrive with the new logger, maybe in the next week or two.

**Comment by karloscarweber** [comment]
Update on this. Camping 3.2 has been released  but didn't include default race support for this. Camping 3.3 will have this fix and I'm actively working on it.

**Comment by karloscarweber** [comment]
@waghanza This should be fixed in the latest version of Camping: (3.2.4). This change required an update from an upstream dependency, **kdl**, which is live. We now depend on that updated dependency. You may also notice other small errors resolved.
